src/NuProj.Targets/NuProj.targets

```diff
@@ -148,7 +150,7 @@
     <ProjectReference>
       <PackageDirectory>Lib</PackageDirectory>
       <TargetFramework></TargetFramework>
-      <Targets>$(ProjectOutputGroups)</Targets>
+      <ProjectOutputGroups>$(ProjectOutputGroups)</ProjectOutputGroups>
```

Comment:
Yes, I think I understand that. But the point is this metadata specifies the targets that should be built in the P2P ref. So I think we want to keep that metadata named that way. But perhaps it's a moot point, because the common.targets set the itemdefinition metadata for Targets. So perhaps we can just delete the ProjectOutputGroups definition here and all references to it.